packages/craco/lib/features/cra-paths/override-utils.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+const { projectRoot } = require("@craco/craco/lib/paths");
```

Comment:
It is bad practice to import from the module you are currently developing. If someone would publish a fork of the repository, he would need to patch all module name mentions.

**Solution**: prefer relative import.

Reply:
Thank you, this is due to an issue caused by using the vscode smart hint, which has been changed to a relative path.